src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -507,10 +513,23 @@ def load_acs_service_principals(config_path):
 
 def _create_kubernetes(resource_group_name, deployment_name, dns_name_prefix, name, ssh_key_value,
                        admin_username="azureuser", agent_count="3", agent_vm_size="Standard_D2_v2",
-                       location=None, service_principal=None, client_secret=None):
+                       location=None, service_principal=None, client_secret=None, windows=False, admin_password=''):
     from azure.mgmt.resource.resources.models import DeploymentProperties
     if not location:
         location = '[resourceGroup().location]'
+    windows_profile = None
+    os_type = 'Linux'
+    if windows:
+        if len(admin_password) == 0:
+            raise CLIError('--admin-password is required.')
```

Comment:
The upcoming check of `<6` is good enough, so I suggest we just remove this `if`